packages/core/src/commands/snapshot-dump.ts

```diff
@@ -35,11 +38,12 @@ export class Command extends Commands.Command {
         this.definition
             .setFlag("token", "The name of the token.", Joi.string().default("ark"))
             .setFlag("network", "The name of the network.", Joi.string().valid(...Object.keys(Networks)))
+            .setFlag("codec", "The name of the codec.", Joi.string())
             .setFlag("skipCompression", "Skip gzip compression.", Joi.boolean())
-            .setFlag("trace", "Dumps generated queries and settings to console.", Joi.boolean())
+            // .setFlag("trace", "Dumps generated queries and settings to console.", Joi.boolean())
```

Comment:
Is this now unused or will it be implemented?

Reply:
It is unused. I commented out unused flags.